lib/Email/Stuffer/TestLinks.pm

```diff
@@ -0,0 +1,77 @@
+package Email::Stuffer::TestLinks;
+
+use strict;
+use warnings;
+
+our $VERSION = '0.010';
+
+use Test::Most;
+use Mojo::UserAgent;
+use Email::Stuffer;
+use Class::Method::Modifiers qw/ install_modifier /;
+
+=head1 SYNOPSIS
+
+    use Email::Stuffer::TestLinks;
+
+=head1 NAME
+
+Email::Stuffer::TestLinks - validates links in HTML emails sent by Email::Stuffer>send_or_die()
+
+=head1 DESCRIPTION
+
+When this module is included in a test, it parses HTML links (<a href="xyz"...) in every email sent through Email::Stuffer->send_or_die(). Each URI must get a successful response code (200 range) and the returned pagetitle must not contain 'error' or 'not found'.
```

Comment:
Probably re-wrap to 80-chars (e.g. for https://metacpan.org/source/Email::Stuffer::TestLinks)

Reply:
Done